examples/HotPotQA/config.py

```diff
@@ -25,9 +25,8 @@ def load_data():
     devset = [formatting(x) for x in dataset.dev]
     return trainset, valset, devset # training and evaluation data
 
-from cognify.hub.search import default
-search_settings = default.create_search(
+from cognify.hub.search import qa
```

Comment:
where's this qa defined?

Reply:
It's in the search hub. That's where we have qa, codegen, etc.